src/main/kotlin/reposfinder/utils/files.kt

```diff
@@ -0,0 +1,19 @@
+package reposfinder.utils
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import java.io.File
+
+fun String.readToken(): String? {
+    val token = File(this).readText()
+    if (token.isEmpty()) {
+        println("File token is empty!")
```

Comment:
It's not good to print somewhere. Users can't handle this input. It's better to use logging. Or in this case, I suggest throwing an error, which would be caught in main and shown to the user.

Reply:
fixed